enarx-keep-sgx-shim/src/handler.rs

```diff
@@ -208,6 +208,7 @@ impl<'a> Handler<'a> {
     /// Proxy an exit() syscall
     ///
     /// The optional `code` parameter overrides the value from `aex`.
+    #[allow(unused_must_use)]
```

Comment:
We don't want to allow this on the whole function. Just the `self.proxy()` invocation. If for some reason you can't put the attribute on the invocation itself, put it on the `loop {}`.